include/wtlgo/Network.hpp

```diff
@@ -6,6 +6,8 @@
 #include <vector>
 #include <map>
 
+#include "Logger.hpp"
```

Comment:
```suggestion
\\ or?
#include <Logger.hpp>
```

Please, edit `CMakeLists.txt` accordingly.